pages/further-reading.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+  <div id="main-content" class="text-wrapper">
+    <h1 class="text-center">Further Reading</h1>
+    <p class="text-justify">
+      The following list of works, curated by Professor Julian Goodare, a
```

Comment:
It might be good to link to Julian's ed.ac.uk profile here: https://www.ed.ac.uk/profile/julian-goodare

Reply:
good idea, done thank you

Reply:
do you think all the links should open in a new tab or fine as is?

Reply:
Ideally they'd open in a new tab as they are external but we'd need to add some kind of visual indicator to show this is an external link then go through the whole site to add to others. I don't think we have time for this so best leave it for now

Reply:
oh I see you've made it a new tab/browser link. That's fine as well. Happy to merge.